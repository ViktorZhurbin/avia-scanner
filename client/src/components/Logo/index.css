.container {
    width: 68px;
    height: 36px;
    cursor: pointer;
}
