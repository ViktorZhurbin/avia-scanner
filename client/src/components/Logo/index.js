import React from 'react';
import SVGInline from 'react-svg-inline';
import cl from 'classnames/bind';

import headerLogo from '../../assets/headerLogo.svg';

import styles from './index.css';

const cx = cl.bind(styles);

const Logo = () => (
    <div className={cx('container')}>
        <SVGInline svg={headerLogo} />
    </div>
);

export default React.memo(Logo);
