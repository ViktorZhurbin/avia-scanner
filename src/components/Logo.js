import React from 'react';
import SVGInline from 'react-svg-inline';

import logoSVG from '../assets/main-logo.svg';

export default () => (
    <div>
        <SVGInline svg={logoSVG} />
    </div>
);
