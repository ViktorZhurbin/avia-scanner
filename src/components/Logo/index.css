.container {
    width: 76px;
    height: 38px;
    cursor: pointer;
}
