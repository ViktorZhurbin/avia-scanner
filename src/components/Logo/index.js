import React from 'react';
import SVGInline from 'react-svg-inline';
import cl from 'classnames/bind';

import logoSVG from '../../assets/main-logo.svg';

import styles from './index.css';

const cx = cl.bind(styles);

const onClick = () => {
    window.history.pushState('', '', '/');
};

const Logo = () => (
    // eslint-disable-next-line
    < div
        className={cx('container')}
        onClick={onClick}
    >
        <SVGInline svg={logoSVG} />
    </div>
);

export default Logo;
