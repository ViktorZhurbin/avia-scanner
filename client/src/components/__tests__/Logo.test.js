import React from 'react';
import { shallow } from 'enzyme';

import Logo from '../Logo';

describe('Render Logo', () => {
    it('render correctly', () => {
        const component = shallow(
            <Logo />,
        );
        expect(component).toMatchSnapshot();
    });
});
