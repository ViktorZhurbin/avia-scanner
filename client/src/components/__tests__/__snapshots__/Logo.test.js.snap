// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Logo render correctly 1`] = `
<div
  className="container"
>
  <SVGInline
    classSuffix="-svg"
    cleanup={Array []}
    cleanupExceptions={Array []}
    component="span"
    svg="test-file-stub"
  />
</div>
`;
